kaznet/apps/ona/tasks.py

```diff
@@ -194,3 +195,28 @@ def task_sync_deleted_projects(usernames: list):
     checks for deleted projects and syncs them
     """
     sync_deleted_projects(usernames=usernames)
+
+
+# pylint: disable=not-callable
+@celery_task(name="task_check_if_users_can_submit_to_form")
+def task_check_if_users_can_submit_to_form(xform_id):
+    """
+    Check if users can submit to the form
+    """
+    try:
+        xform = XForm.objects.get(pk=xform_id)
+    except XForm.DoesNotExist:  # pylint: disable=no-member
+        pass
```

Comment:
Could we consider adding logging? Probably this would help in doing some debugging in the future.

Reply:
This is a good catch.  Might not do it in this PR, but I created an issue to address this: https://github.com/onaio/kaznet-web/issues/314